config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -167,4 +167,94 @@
   <!-- We use negative suppression to check workflow files only. -->
   <suppress id="workflowJobStepSpacing" files="^.((?!github[\\/]workflows[\\/]).)*$" />
 
+  <!-- Examples macro suppressions -->
```

Comment:
We can create one issue per section here to get macros in place for each section here if we agree to this.